lib/cirneco/api.rb

```diff
@@ -109,5 +109,28 @@ def get_media(doi, options={})
       end
       response
     end
+  
+    def update_rest_doi(doi, options={})
+      return OpenStruct.new(body: { "errors" => [{ "title" => "JWT or Username or password missing" }] }) unless options[:jwt].present? || (options[:username].present? && options[:password].present?)
+
+      api_url = options[:sandbox] ? 'https://api.test.datacite.org' : 'https://api.datacite.org'
```

Comment:
Can we move this to somewhere more central, you repeat it in every function.
Presumably there is some global config somewhere.